LPC Projects/CAN_driver/src/can.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * CAN driver that implements basic CAN_init, CAN_send, and CAN_receive
+ * User has to write their own CAN_tx, CAN_rx, and CAN_error functions
```

Comment:
Is the intention that CAN_receive is called by the user-defined CAN_rx callback?  And likewise with CAN_send?

Reply:
`CAN_receive` should be renamed to `CAN_setup_receiver`
`CAN_send` should be renamed to `CAN_send_msg`
`CAN_tx`, `CAN_rx`, `CAN_error` should be renamed to `*_cb`, since they're callback functions